src/collection.js

```diff
@@ -248,7 +249,7 @@ function importChange(transaction, remote, localFields) {
     if (local._status === "deleted") {
       return { type: "skipped", data: local };
     }
-    if (isIdentical) {
+    if (isIdentical || strategy === Collection.strategy.PULL_ONLY) {
```

Comment:
I think we can even skip the check to see if records are identical if we're in `PULL_ONLY`. Not sure it's worth implementing though..

Reply:
Well, we must now if the record was there or if it was updated though. Maybe just compare timestamps instead?

Reply:
I don't understand. According to the code you wrote, we don't actually need to know if the record was updated.

Reply:
Actually, I was confused because the integration tests were implemented in a weird way (synced records were manually created on the server and the client, I replaced this by a sync). After rewroting the test setup I could skip the records comparison as expected :) Thanks for suggestion!